oic.r.cred.raml

```diff
@@ -55,8 +55,7 @@ traits:
                     "subjectuuid": "e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9",
                     "roleid": {
                       "authority": "484b8a51-cb23-46c0-a5f1-b4aebef50ebe",
-                      "roles": [
-                        "SOME_STRING", "SOME_STRING"
+                      "role": "SOME_STRING"
                       ]
```

Comment:
Whoops, good eye, Mitch! Of course I had to miss one to make sure you were paying attention. Will update momentarily.